mirage/scenarios/dashboard.ts

```diff
@@ -101,39 +101,38 @@ export function dashboardScenario(server: Server, currentUser: ModelInstance<Use
 
     const boxAccount = server.create('authorized-storage-account', {
         displayName: 'My Box Account',
-        scopes: ['write'], // TODO: This should be a from an enum?
+        // scopes: ['write'], // TODO: This should be a from an enum?
```

Comment:
What's happening here?

Reply:
It looks like I commented these out to address later (`scopes` should have been renamed `authorizedCapabilities`), and forgot to address them. Fixing these now